config

```diff
@@ -0,0 +1,12 @@
+{
+    "netbox": {
+        "netbox_url": "",
+        "netbox_token": "",
+        "ucs_hostname_format": "",
+        "regions": [],
+        "tenant": "",
+        "query": "",
+        "status": ""
+    }
+
+}
```

Comment:
Make this a yaml, way easier handling and also name it like this: `config.yaml`

Reply:
Created Yaml, please check.

Reply:
please remove this file

Reply:
Way better, thanks 👍